Dispatchfile

```diff
@@ -26,7 +26,7 @@ dind_task("publish",
               export GIT_REMOTE_URL=https://mesosphere:${GITHUB_USER_TOKEN}@github.com/mesosphere/charts.git
               git config --global user.name 'Dispatch CI'
               git config --global user.email '56653984+d2iq-dispatch@users.noreply.github.com'
-              make publish HELM_VERSION=v2.16.9
+              make publish
```

Comment:
@faiq you set this explicitly before https://github.com/mesosphere/charts/pull/740

Will this change break anything in dispatch since we're trying to publish v3 charts now?

Reply:
This is just running "make publish" and specifying an old helm version. We don't want this set here, we want it set in the Makefile.